packages/react-scripts/package.json

```diff
@@ -44,7 +44,9 @@
     "eslint-plugin-import": "2.16.0",
     "eslint-plugin-jsx-a11y": "6.2.1",
     "eslint-plugin-react": "7.12.4",
+    "eslint-plugin-react-hooks": "1.5.0",
     "file-loader": "3.0.1",
+    "fork-ts-checker-webpack-plugin-alt": "0.4.14",
```

Comment:
What's this?

Reply:
Yeah this should not be here. Rebase gone wrong?